README-cert.md

```diff
@@ -0,0 +1,84 @@
+Cert module
```

Comment:
The README still needs work - I'll populate it once module options etc are finalized